api/envoy/api/v2/cluster.proto

```diff
@@ -839,13 +839,13 @@ message Cluster {
 message LoadBalancingPolicy {
   message Policy {
     // Required. The name of the LB policy.
-    string name = 1;
+    string name = 1 [deprecated = true];
```

Comment:
I don't think this needs to be deprecated in v2.  It doesn't really make sense to do this unless we're also adding the new field to v2, and I don't see any need for that.

Instead, these fields should be kept in v3 and marked as deprecated there.

Reply:
Done.

Reply:
Thanks!  The API changes look great here, but I'll let Lizan approve.